PV_Excess_Control/blueprints/automation/pv_excess_control.yaml

```diff
@@ -252,8 +381,10 @@ blueprint:
           unit_of_measurement: A
     actual_power:
       name: "Appliance actual power sensor"
-      description: "Sensor which contains the **current power consumption** of the appliance in watts.\n
-      If this is left empty (not recommended), the *typical current draw* will be used instead."
+      description: >
+        Sensor which contains the **current power consumption** of the appliance in watts.
+
+        If this is left empty (not recommended), the *typical current draw* will be used instead."
```

Comment:
The last symbol in this line can be removed (")

Reply:
Yes, you are right. It is corrected now